config/locales/en.yml

```diff
@@ -1131,6 +1131,31 @@ en:
       link: "Visit here to give them access: %{link}"
       subject: "%{facility} - User Requested Training"
 
+  upcoming_offline_reservation_mailer:
+    subject: Regarding your upcoming reservation for %{instrument}
+    send_offline_instrument_warning:
+      body:
+        html: |
+          <p>
+            %{product}, the instrument you
+            <a href="%{reservation_url}">have reserved for %{reservation_time}</a>,
```

Comment:
If we use the `html` text helper method, we can use markdown in the locale. No need for the `<p>` tags and cleaner links.